src/main/java/ca/on/oicr/pineryreports/util/SampleUtils.java

```diff
@@ -158,6 +158,24 @@ public static boolean isNonIlluminaLibrary(SampleDto dilution) {
     return dto -> isNonIlluminaLibrary(dto);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)